velox/docs/develop/testing/spark-query-runner.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Spark Query Runner
+==================
+
+Introduction
+------------
+
+The Spark Query Runner is a tool designed to facilitate the testing of Velox.
+It achieves this by executing SQL queries on Apache Spark and comparing the results
+with those produced by Velox. It helps ensure the correctness of Velox's computing
+against Spark and provides a method for identifying potential issues in Velox's
+implementation. Currently Spark-3.5.1 is used in Velox.
+
+How It Works
+------------
+
+The Spark Query Runner operates by executing given SQL queries on both Spark and Velox.
+The results from both systems are then compared to check for any differences.
+If the results match, it indicates that Velox is producing the correct output.
+If the results differ, it suggests a potential issue in Velox that needs to be
+investigated.
+
+Since Spark 3.4, Spark Connect has introduced a decoupled client-server architecture
+for Spark that allows remote connectivity to Spark clusters. From the client
+perspective, Spark Connect mostly behaves as any other gRPC client, which is polyglot
+and and cross-platforms. During execution, the Spark Connect endpoint embedded on the
+Spark Server receives and translates unresolved logical plans into Spark’s logical plan
```

Comment:
> receives and translates unresolved logical plans into Spark’s logical plan

Aren't we sending SQL? If so, this statement is confusing as it suggests that we must send a logical plan.

Reply:
Changed as below. Thanks.

> receives and parses queries into Spark’s logical plan operators